types/evse_security.yaml

```diff
@@ -260,4 +263,22 @@ types:
         description: The requested info
         type: object
         $ref: /evse_security#/CertificateInfo
+  GetCertificateFullInfoResult:
+    description: Response to the command get_all_valid_certificates_info
+    type: object
+    required:
+      - status
+      - info
+    properties:
+      status:
+        description: The status of the requested command
+        type: string
```

Comment:
Is it really a string?

Reply:
Yes, string: https://github.com/EVerest/everest-core/blob/b11c690bc2158b713a16c5e81be73341a94cfd74/types/evse_security.yaml#L87